source/fundamentals/authentication.txt

```diff
@@ -1 +1,208 @@
-.. _csharp-authentication-mechanisms:
\ No newline at end of file
+.. _csharp-authentication-mechanisms:
+
+=========================
+Authentication Mechanisms
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to authenticate with MongoDB using each
+**authentication mechanism** available in the {+mongo-community+}.
+Authentication mechanisms are processes by which the driver and server confirm
+identity and establish trust to ensure security before connecting.
+
+You can use the following authentication mechanisms with the latest version
+of {+mongo-community+}:
+
+- :ref:`csharp-scram-sha-256`
+- :ref:`csharp-scram-sha-1`
+- :ref:`csharp-x509`
+
+To authenticate using ``Kerberos`` or ``LDAP``, see the
+:ref:`csharp-enterprise-authentication-mechanisms` guide. For more information
+on establishing a connection to your MongoDB cluster, see the
+:ref:`csharp-connection` guide.
+
+Specify an Authentication Mechanism
+-----------------------------------
+
+You can specify your authentication mechanism and credentials when connecting to
+MongoDB using either of the following:
+
+- A connection string
+- A ``MongoCredential`` factory method
+
+A **connection string**, also known as a **connection uri**, is a string that
+tells the driver how to connect to a MongoDB deployment and how to behave while
+connected.
+
+The ``MongoCredential`` class contains factory methods for each of the supported
+authentication mechanisms. 
+
+Mechanisms
+----------
+
+The following examples specify authentication mechanisms using the following
+placeholders:
+
+- ``username`` - Your MongoDB username
+- ``password`` - Your MongoDB user's password
+- ``hostname`` - The network address of your MongoDB server, accessible by your client
+- ``port`` - The port number of your MongoDB server
+- ``authenticationDb`` - The MongoDB database that contains your user's authentication
+  data. If you omit this parameter, the driver uses the default value ``admin``.
+
+Select the :guilabel:`Connection String` or :guilabel:`MongoCredential` tab to
+see the corresponding syntax for specifying your authentication mechanism.
```

Comment:
After reading this, I expected to see the tabs right below it but there was another section.

Reply:
But then this same text is replicated in the Default section -- should this be here?

Reply:
I initially intended it to just be a statement that applies to the whole page. But I agree it's a little confusing here. Removed it from here and left it in the other areas